backend/src/schema/resolvers/users.spec.js

```diff
@@ -119,12 +100,12 @@ describe('users', () => {
         await expect(client.request(mutation, variables)).resolves.toEqual(expected)
```

Comment:
don't know what this is testing...

Reply:
I think it is testing that you cannot alter other user accounts, only your own.